deepchecks/vision/suites/default_suites.py

```diff
@@ -69,8 +231,66 @@ def integrity_validation(**kwargs) -> Suite:
     return data_integrity(**kwargs)
 
 
-def data_integrity(**kwargs) -> Suite:
-    """Create a suite that includes integrity checks."""
+def data_integrity(image_properties: List[Dict[str, Any]] = None,
+                   n_show_top: int = 5,
+                   label_properties: List[Dict[str, Any]] = None,
+                   **kwargs) -> Suite:
+    """
+    Create a suite that includes integrity checks.
+
+    List of Checks:
+        .. list-table:: List of Checks
+           :widths: 50 50
+           :header-rows: 1
+
+           * - Check Example
+             - API Reference
+           * - :ref:`plot_vision_image_property_outliers`
+             - :class:`~deepchecks.vision.checks.data_integrity.ImagePropertyOutliers`
+           * - :ref:`plot_vision_label_property_outliers`
+             - :class:`~deepchecks.vision.checks.model_evaluation.LabelPropertyOutliers`
+
+    Parameters
+    ----------
+    image_properties : List[Dict[str, Any]], default: None
+        List of properties. Replaces the default deepchecks properties.
+        Each property is dictionary with keys 'name' (str), 'method' (Callable) and 'output_type' (str),
+        representing attributes of said method. 'output_type' must be one of:
+        - 'numeric' - for continuous ordinal outputs.
+        - 'categorical' - for discrete, non-ordinal outputs. These can still be numbers,
+          but these numbers do not have inherent value.
+        For more on image / label properties, see the :ref:`property guide </user-guide/vision/vision_properties.rst>`
+    n_show_top : int , default: 5
```

Comment:
only for outliers? (similar comment to above)

Reply:
Here both checks are outlier detection. changed to `samples` instead